shenyu-integrated-test/shenyu-integrated-test-spring-cloud/admin-config/schema.sql

```diff
@@ -0,0 +1,21 @@
+-- noinspection SqlNoDataSourceInspectionForFile
```

Comment:
Please move this file to /src/test/resources/admin-config/

According lastttttt meeting, we think this would be better